scss/components/_filter-selection.scss

```diff
@@ -79,10 +79,10 @@
 
 .filter-overview {
   &__error {
-    border-left: thick solid $poppy;
+    border-left: thick solid $ruby-red;
```

Comment:
`thick` should appear after `solid`.
>Always declare the border-style property before the border-width property. An element must have borders before you can set the width.
I'd prefer that `thick` was a known value. 
### Mac
Chrome `thick` = 5px
Firefox `thick` = 5px
Safari `thick` = 5px

### PC
Chrome `thick` = 3.333px
IE = ?? 
Edge `thick` = 5px